manifests/profile/aleph/perl.pp

```diff
@@ -0,0 +1,30 @@
+
+# Copyright (c) 2021 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# nebula::profile::aleph::perl
+#
+# Install perl dependencies for aleph/makersmark applications
+#
+# @example
+#   include nebula::profile::aleph::perl
+class nebula::profile::aleph::perl () {
+
+  ensure_packages([
```

Comment:
Sort alpha

Reply:
Not sure what you mean by this one?

Reply:
Sorted alphabetically.   Should be all set.